### ISSUE TITLE
Should `axes` be required parameter for layerNormalization build method?

### ISSUE THREAD

**Issue by huningxin** [webmachinelearning/webnn#487]
(This was raised by @wacky6 at Chromium [CL-5068275](https://chromium-review.googlesource.com/c/chromium/src/+/5068275/comment/29d44c65_5ca8d830/) review)

In proposal [Add support for operations needed for well-known transformers ](https://github.com/webmachinelearning/webnn/pull/478#discussion_r1390547694), the `axes` of `layerNormalization` operator is defined as an optional member of `MLLayerNormalizationOptions` dictionary with default value [1, 2, 3].

> axes, of type sequence<unsigned long>
>   A sequence of [unsigned long](https://webidl.spec.whatwg.org/#idl-unsigned-long). The indices to the input dimensions to reduce. When this member is not present, it is assumed to be [1,2,3] that is, the reduction for the mean and variance values are calculated across all the input features for each individual sample in the batch.

@wacky6 mentioned TensorFlow's [layerNormalization](https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization) defaults to the last dimension (axis=-1) while PyTorch's [LayerNorm](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html) requires the `normalized_shape` to be passed.

> In terms of API design, seems axes are required for this op to function. Should this be moved outside of options?

> layerNormalization(sequence<[EnforceRange] unsigned long> axes, options)

/cc @wchao1115 @fdwr

**Comment by fdwr** [comment]
**Ningxin**: Different ML libraries having different defaults for `axes` is a little concerning, as it's tough to favor one default over the other. Given the layer in the hierarchy that WebNN targets (lower-level backend and less end-user facing), I generally favor being explicit over being implicit. Frameworks will know their own expected behavior.
